test/helpers/views/supplementary/PackageInformationMessages.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package helpers.views.supplementary
```

Comment:
These files may be really useful. I think you should give a brief informal presentation (5-10 min) to the team of how and why we should use this.

Reply:
This would help us understand much better the value of what you have done and it could raise interesting questions.

Reply:
I just used to avoid errors while testing the messages from file - they are used for now in tests for controllers/views.